jenkins/job-configs/kubernetes-jenkins/clean-ci-node-e2e.yaml

```diff
@@ -0,0 +1,18 @@
+- job:
+    name: 'clean-ci-node-e2e'
+    description: 'Clean up leaked node e2e resources. Test owner: test-infra maintainers.'
+    node: ''
```

Comment:
Nice catch!

I'm going to run it on a light agent, added a TODO in attach_agent.py to create an easier to understand name.

I'd like to avoid running things on the master unless necessary (easier to update master, master less likely to crash, more performant, etc)